src/libs/utils.ts

```diff
@@ -18,3 +22,24 @@ export function sortRecordsByNameLabel(
 export function escapeRegExp(string: string) {
   return string.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
 }
+
+export function getFilterIcon(filter: Filter | undefined) {
+  if (!filter) {
+    return;
+  }
+
+  if (filter.icon) {
+    return filter.icon;
+  }
```

Comment:
```suggestion
  if (filter.icon !== undefined) {
    return filter.icon;
  }
```
Explicit expression to avoiding confusion with a boolean.